lib/vsdata.cpp

```diff
@@ -628,10 +627,11 @@ void Key_Mod_a_Pressed(GLenum state)
 
       int precision = prompt<int>("Enter precision (4): ",
                                   &default_precision,
-                                  [](int p){ return p>=0; });
-      char format = prompt<char>("Enter format [(d)efault, (f)ixed, (s)cientific] (d): ",
-                                 &default_format,
-                                 [](char c){ return c=='d' || c=='f' || c=='s'; });
+      [](int p) { return p>=0; });
+      char format =
+         prompt<char>("Enter format [(d)efault, (f)ixed, (s)cientific] (d): ",
+                      &default_format,
+      [](char c) { return c=='d' || c=='f' || c=='s'; });
```

Comment:
Here too 😇

Reply:
☝️

Reply:
fixed in https://github.com/GLVis/glvis/pull/299/commits/ed619725e66331dabd685d65e403b12b78747ea7

Reply:
nope

Reply:
I can run the styler, or keep it how it was, but if you want something else I don't know what it is.

Reply:
You must fight it until it does not change the code, otherwise it will not pass the checks on master.